cfn-templates/data-exports-aggregation.yaml

```diff
@@ -502,6 +527,65 @@ Resources:
         RefreshCadence:
           Frequency: "SYNCHRONOUS"
 
+  # FOCUS
+
+  ## Deploy Data Export natively via CFN resource if we are regions that support native CFN
```

Comment:
Still need to change wording to "## Deploy Data Export natively via CFN resource in regions that support native CFN"